index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>mobile version</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a class="logo"> My Logo </a>
+      <nav>
+         <button type="button" class="btn">
+          <img alt="nav" class="menu" src="./images/Union.png" />
+         </button>
+      </nav>
+    </header>
+  
+    <section class="flex-container">
+      <div class="text">
+        <h1>I'm Loic<br />Glad to see<br />you</h1>
+      </div>
+      <p class="paragraph">
+        I'm software developer! I can help you<br />to build a product, feature
+        or website look<br />through some of my work and<br />
+        experience! If you like what you see and<br />have a project you need to
+        coded, don't<br />hesitate to contact me.
+      </p>
+      <h4 class="connect">LET'S CONNECT</h4>
+      <div class="social">
+        <img alt="twitter" src="./images/twitter.png" />
+        <img alt="linkedin" src="./images/linkedin.png" />
+        <img alt="medium" src="./images/medium.png" />
+        <img alt="github" src="./images/github.png" />
+        <img alt="vec" src="./images/vec.png" />
+      </div>
```

Comment:
- [ ] The list of icons should use a ```ul``` tag to group the icons, and ```li``` per icon, then each icon should have a link (```a``` tag) e.g. 
   ```
   <ul class="social">
        <li><a href="#><img alt="twitter" src="./images/twitter.png" /></a></li>
   ...
   ...
   </ul>
   ```